privaterelay/sp3_plans.py

```diff
@@ -0,0 +1,250 @@
+from functools import lru_cache
```

Comment:
When we're moved to SP3, we can delete the existing `privaterelay.plans` module.